bbp/hpc/syntool/default.nix

```diff
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
     name = "syn-tool-${version}";
-    version = "0.1-dev201709";
+    version = "0.1-dev201805";
 
 
     src = fetchgitPrivate {
         url = "ssh://bbpcode.epfl.ch/hpc/synapse-tool";
-        rev = "f02fff48867b3acd9162515fdc224f1b8f48e9df";
-        sha256 = "15iarkk9fsama2mdj0ji5zcknf8r9z6490fcinznrn8fvv1lnd0z";
+        rev = "6ef3eb401f3fca5badfaa77d40ae6728970c6cba";
+        sha256 = "06iwz15f67nb01y904xwy9j52qk8bhflclkyi4b488k4qh91cakf";
```

Comment:
Do you need to bump syntool?

Reply:
The converters require MPI additions that Fernando added to that branch.